HealthVault/Classes/Models/Generated/MHVActionPlan.h

```diff
@@ -29,13 +29,22 @@
 #import "MHVActionPlanTask.h"
 #import "MHVObjective.h"
 #import "MHVModelBase.h"
+#import "MHVEnum.h"
 
 
 @protocol MHVActionPlan
 @end
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface MHVActionPlanCategoryEnum : MHVEnum
++(MHVActionPlanCategoryEnum *) Unknown;
```

Comment:
1. When using the dynamic enums, I believe that you don't need to specify unknown explicitly; The super class should add that for you.
2. Can we remove the space between the ending parenthesis and the name of the enum?
3. We should prefix the enum method name with MHV. We had an issue with the HVI app getting rejected because of collisions with other iOS framework properties.